common/app/model/RelatedContent.scala

```diff
@@ -25,8 +25,17 @@ case class RelatedContent (
 
 object RelatedContent {
   def apply(parent: ContentType, response: ItemResponse): RelatedContent = {
-    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this file.
-    val items = response.storyPackage.map { item =>
+    // It's misleading to use storyPackage here rather than relatedContent. A tidy up should rename this object
```

Comment:
Yes, `RelatedContent`could do with another look, now that the content model is a bit more stable.